src/users/routes/usersRouter.js

```diff
@@ -1,20 +1,18 @@
 import { Router } from 'express';
-
 import { userController } from '../controllers';
```

Comment:
import controllers directly and avoid importing from root/index

When you import from index it's gonna require the whole dependency tree (even for those modules that are not needed for user routes).

So delete all folder indexes and import modules by full path